Lesson15/Akan/script.js

```diff
@@ -0,0 +1,407 @@
+/*
+4. Extract Domain
+   - Define a function `getDomain(email)` that uses `slice` or `substring` to
+     extract everything after '@'.
+   - Log the domain to the console.
+   - Example: "user@example.com" -> "example.com"
+*/
+
+function getDomain(email) {
+  let domain = email.slice(email.indexOf('@') + 1);
+  console.log(domain);
+}
+
```

Comment:
The code above (getDomain function) is a function definition in JS. After defining a function, it's important to call it, otherwise the code will not execute. Make sure that you test functions after adding them like we did in the lessons to spot any errors.

```suggestion
getDomain(test@gmail.com);
```